modules/admin_manual/pages/configuration/server/occ_commands/app_commands/admin_audit_commands.adoc

```diff
@@ -0,0 +1,42 @@
+= Auditing
+
+Marketplace URL: {oc-marketplace-url}/apps/admin_audit[Auditing]
+
+Tracks various activities and actions of your users and admins. For details please see the
```

Comment:
```suggestion
Tracks various activities and actions of your users and admins. For details, please see the
```

Reply:
@EParzefall you are the Komma-Expert, I am unsure.

Reply:
@jnweiger  It's an optional comma. I like it.